pandas/core/generic.py

```diff
@@ -4306,7 +4306,7 @@ def __delitem__(self, key) -> None:
     # Unsorted
 
     @final
-    def _check_inplace_and_allows_duplicate_labels(self, inplace):
+    def _check_inplace_and_allows_duplicate_labels(self, inplace: bool_t):
```

Comment:
nice 😄

Reply:
It was `bool`, had to change it to `bool_t`

Reply:
yeah that's what I meant, I presume the other hook caught this one?

Reply:
mypy made me change it :)